examples/rest-parameters/rest_parameters.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
+
+function foo(int n, string... s) {
+    io:println(n);
+    io:println(s[0]);
+    io:println(s[1]);
+    io:println(s[2]);
+    io:println(s);
+    io:println(s is string[]);
```

Comment:
Isn't it better if we move this to after line 4.
```
    io:println(n);
    io:println(s is string[]);
    io:println(s[0]);
    io:println(s[1]);
    io:println(s[2]);
    io:println(s);
```

Reply:
As we have few number of lines and just `println`s. It would be ok. if we have any logical grouping we can split